### DISCUSSION TITLE
两栏情况下，文章如何显示更宽

### DISCUSSION THREAD

**Discussion by leonyan18** [ppoffice/hexo-theme-icarus#1164] [Q&A]
version：5.1.0
如何在两栏情况下，文章内容显示范围太少了，希望更加宽一点
过去方法是在主题文件夹下修改
现在目前已知方法都是去node_modules里面修改的 有没有更加优雅的方法

**Comment by am-abudu** [comment]
你可能需要 #696

**Comment by realzza** [comment]
首先改变页面content宽度，修改`themes/icarus/include/style/responsive.styl`
其次修改栏目比例，参考[官网FAQ](https://ppoffice.github.io/hexo-theme-icarus/uncategorized/faq/#:~:text=.yml%20extension.-,Layout,-How%20do%20I)-Layout

**Reply by leonyan18** [reply]
主要是5.0，themes下没有icarus了

**Reply by am-abudu** [reply]
你依然可以将icarus放在themes下
